src/PropertyInfo/TypeInfoParsers/NativeTypeInfoParser.php

```diff
@@ -19,11 +17,8 @@
  *
  * @author Mihai Stancu <stancu.t.mihai@gmail.com>
  */
-abstract class NativeTypeInfoParser implements TypeInfoParserInterface
+trait NativeTypeInfoParser
```

Comment:
Refactoring done but PHP7 tests fail because of [this](https://github.com/sebastianbergmann/phpunit/issues/1765) `PHPUnit 5<` issue.

If the tests are run with `PHPUnit 5` they work.